Source/Device.generated.swift

```diff
@@ -1100,7 +1100,7 @@ extension Device {
   /// Return the root url
   ///
   /// - returns: the "/" url
```

Comment:
Please compile the project in Xcode so the change to this line in your gyb file propagates to the `.generated.swift` file.

Reply:
Already done.